template/package.json

```diff
@@ -93,6 +93,9 @@
     "vue-style-loader": "^3.0.1",
     "vue-template-compiler": "^2.3.3",
     "webpack": "^2.6.1",
+    {{#cdn}}
+    "webpack-cdn-plugin": "^1.3.4",
```

Comment:
I've tested running `vue init van-nguyen/webpack` seems there is no empty lines, maybe it's smart enough to detect these things.